### PR TITLE
✨(feature): Added a view activity page for a parent

### PR DESCRIPTION
**Pull request by d-azmanov**

### CODE REVIEW

**Review by priyo5 — APPROVED** [review]

File: libs/parent-profile/feature/src/lib/parent-profile.component.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
     margin-right: auto;
     background-color: #69A88DB2;
     border-radius: 16px;
-    margin-bottom: 60px;
+    margin-bottom: 30px;
```
Comment (PxttytheThxtty) [inline libs/parent-profile/feature/src/lib/parent-profile.component.scss:18]: This is the wrong file.

**Review by PxttytheThxtty — CHANGES_REQUESTED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

File: libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss [hunk 2]
```diff
@@ -0,0 +1,206 @@
+ion-content{
+    --ion-background-color:#EFEFEF;
```
Comment (ricecakesreece) [inline libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss:2]: var(--ion-color-medium)

File: libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+#nav{
+    display: block;
+    width: 100%;
+    height: 60px;
+    background-color: #176560;
```
Comment (ricecakesreece) [inline libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss:9]: var(--ion-color-secondary)

File: libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    align-items: center;
+    flex-direction: column;
+    text-align: right;
+}
+
+// Button styling
+.activity-details input{
+    width: 411px;
+    height: 62px;
+    background: #F9F9F9;
```
Comment (ricecakesreece) [inline libs/parent-view-activity/feature/src/lib/parent-view-activity.component.scss:64]: var(--ion-background-color)

**Review by ricecakesreece — APPROVED** [review]
Big approve

**Review by PxttytheThxtty — APPROVED** [review]

**Pull request merged**